cost/azure/savings_plan/utilization/azure_savings_plan_utilization.pt

```diff
@@ -0,0 +1,501 @@
+name "Azure Savings Plan Utilization"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications when Azure Savings Plan Utilizations are identified. NOTE: These Utilizations are generated by Azure. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/azure/savings_plan/utilization) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
```

Comment:
Should be reworded to avoid mentioning email notifications and be more clear. Example:

Reports the utilization of Azure Savings Plans within the Azure estate.